dist/vue-toastr.umd.js

```diff
@@ -705,13 +705,13 @@
   const __vue_script__$2 = script$2;
 
   /* template */
-  var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}})}),1)}),0)};
+  var __vue_render__$2 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',_vm._l((_vm.list),function(toasts,position){return _c('div',{key:position,class:'toast-container ' + position},_vm._l((toasts),function(toast,index){return _c('toast',{key:index,attrs:{"data":toast}},[_vm._t("default")],2)}),1)}),0)};
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Split this 351 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RYQAaEKZWHEUhII&open=AWz13RYQAaEKZWHEUhII)

[ISSUE_KEY]: AWz13RYQAaEKZWHEUhII